helpers.js

```diff
@@ -35,9 +35,14 @@ export const createInitialVc = async ({issuer, vc} = {}) => {
 // RegExp with bs58 characters in it
 const bs58 =
   /^[123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz]+$/;
+
+const BASE_64URL_REGEX = /^[A-Za-z0-9\-_=]+$/;
```

Comment:
I'll check this against the published spec. Note the pad `=` in there, which I dont think is technically on base64url, but ill double-check